datajunction-clients/python/README.md

```diff
@@ -1,12 +1,20 @@
 # DataJunction Python Client
 
+This is a short introduction into the Python version of the Data Junction (DJ) client.
+For a full comprehensive intro into the DJ functionality please check out [datajunction.io](https://datajunction.io/).
+
 ## Installation
+
 To install:
 ```
 pip install datajunction
 ```
 
-## Examples
+## DJ Client : Basic Access
+
+Here you can see how to access and use the most common Data Junction features.
```

Comment:
Just a small nit: we should eventually settle on a single way to write the full project name across documentation. I see a lot of variations like datajunction, DataJunction, Datajunction, etc.

Reply:
We should... `DataJunction` and `DJ` ?